ansible/roles/stack-sunbird/tasks/content-service.yml

```diff
@@ -5,18 +5,18 @@
   
 - name: Save configurations into an env file
   become: yes
-  template: src=content-service_application.conf dest=/home/deployer/config/application.conf mode=0644
+  template: src=content-service_application.conf dest=/home/deployer/config/content-service.conf mode=0644
 
 - name: Remove old application.conf docker config
   become: yes
-  shell: "docker config rm application.conf"
+  shell: "docker config rm content-service.conf"
   ignore_errors: yes
   
 - name: Save as docker config
   become: yes
-  shell: "docker config create application.conf /home/deployer/config/application.conf"
+  shell: "docker config create content-service.conf /home/deployer/config/content-service.conf"
 
 - name: Deploy content-service
-  shell: "docker service create --with-registry-auth --replicas {{ learner_replicas }} -p 9002:9000  --name content-service --hostname content-service --reserve-memory {{ learner_reservation_memory }} --limit-memory {{ learner_limit_memory }} --limit-cpu {{ learner_limit_cpu }} --reserve-cpu {{ learner_reservation_cpu }} --health-cmd 'wget -qO- content-service:9000/health || exit 1' --health-timeout 3s --health-retries 3  --network application_default --config source=application.conf,target=/home/sunbird/content-service-1.0-SNAPSHOT/config/application.conf,mode=0644  {{hub_org}}/{{image_name}}:{{image_tag}}"
+  shell: "docker service create --with-registry-auth --replicas {{ learner_replicas }} -p 9002:9000  --name content-service --hostname content-service --reserve-memory {{ learner_reservation_memory }} --limit-memory {{ learner_limit_memory }} --limit-cpu {{ learner_limit_cpu }} --reserve-cpu {{ learner_reservation_cpu }} --health-cmd 'wget -qO- content-service:9000/health || exit 1' --health-timeout 3s --health-retries 3  --network application_default --config source=content-service.conf,target=/home/sunbird/content-service-1.0-SNAPSHOT/config/application.conf,mode=0644  {{hub_org}}/{{image_name}}:{{image_tag}}"
```

Comment:
@maheshkumargangula Using learner service variables is not accepted!

Reply:
@keshavprasadms - Done. Please validate.